### DISCUSSION TITLE
插旗awa

### DISCUSSION THREAD

**Discussion by Someone-Yang** [Clipteam/clipcc-gui#46] [闲聊/General]
祝贺 Discussions 功能启用！
终于可以更近距离地跟进更新消息了！
（而网络日常抽风是最令人痛苦的）
哔哩哔哩现已拥有*含推荐的*开放的 **『ClipCC3』** 标签，各位可以关注来自各地UP主，也可以在投稿时添加标签，丰富标签生态，为ClipCC喝彩和贡献内容
愿 ClipCC 更好！

---

来自b站粉丝私信：

> clipcc挺好，就是很少更新

**Comment by SimonShiki** [comment]
支持一下
